Amplify/Amplify.swift

```diff
@@ -63,6 +68,27 @@ public class Amplify {
         }
     }
 
+    @available(iOS 13.0.0, *)
+    public static func enableDevMenu(devMenuDelegate: DevMenuDelegate) {
+        DevMenu = AmplifyDevMenu(delegate: devMenuDelegate)
+        isDeveloperMenuEnabled = true
+    }
+
+    static func isDevMenuEnabled() -> Bool {
+        return isDeveloperMenuEnabled
+    }
+
+    static func getPlugin() -> LoggingCategoryPlugin {
```

Comment:
Where is this method used?

Reply:
This will be used in LoggingCategory.swift while getting the default plugin.